.cirrus.yml

```diff
@@ -133,46 +133,81 @@ windows_task:
+      env:
+        install_packages: nghttp2
+        configure: --without-ssl --enable-debug
+    - name: macOS arm64 libssh2
+      env:
+        install_packages: nghttp2 libssh2
+        prefix: $(brew --prefix libssh2)
+        configure: --without-ssl --enable-debug --with-libssh2
+    - name: macOS arm64 libssh-c-ares
+      env:
+        install_packages: nghttp2 openssl libssh
+        prefix: $(brew --prefix openssl)
+        configure: --enable-debug --with-openssl --with-libssh --enable-ares
+    - name: macOS arm64 libssh
+      env:
+        install_packages: nghttp2 openssl libssh
+        prefix: $(brew --prefix openssl)
+        configure: --enable-debug --with-openssl --with-libssh
+    - name: macOS arm64 c-ares
+      env:
+        install_packages: nghttp2
+        configure: --without-ssl --enable-debug --enable-ares
+    - name: macOS arm64 HTTP only
+      env:
+        install_packages: nghttp2
+        configure: --enable-debug --enable-maintainer-mode --disable-dict --disable-file --disable-ftp --disable-gopher --disable-imap --disable-ldap --disable-pop3 --disable-rtmp --disable-rtsp --disable-scp --disable-sftp --disable-smb --disable-smtp --disable-telnet --disable-tftp --disable-unix-sockets --disable-shared --without-brotli --without-gssapi --without-libidn2 --without-libpsl --without-librtmp --without-libssh2 --without-nghttp2 --without-ntlm-auth --without-ssl --without-zlib
+    - name: macOS arm64 SecureTransport http2
+      env:
+        install_packages: nghttp2
+        configure: --with-secure-transport
+    - name: macOS arm64 OpenSSL http2
+      env:
+        install_packages: nghttp2 openssl
+        prefix: $(brew --prefix openssl)
+        configure: --enable-debug --with-openssl
+    - name: macOS arm64 LibreSSL http2
+      env:
+        install_packages: nghttp2 libressl
+        prefix: $(brew --prefix libressl)
+        configure: --enable-debug --with-openssl
+    - name: macOS arm64 torture
+      env:
+        install_packages: nghttp2 openssl
+        prefix: $(brew --prefix openssl)
+        configure: --enable-debug --disable-shared --disable-threaded-resolver --with-openssl
+    - name: macOS arm64 torture-ftp
+      env:
+        install_packages: nghttp2 openssl
+        prefix: $(brew --prefix openssl)
+        configure: --enable-debug --disable-shared --disable-threaded-resolver --with-openssl
 
   env:
     CIRRUS_CLONE_DEPTH: 10
     MAKE_FLAGS: -j 4
-    CFLAGS: "-Wno-vla -mmacosx-version-min=10.9"
+    CFLAGS: "-Wno-vla -mmacosx-version-min=10.8"
 
   pkginstall_script:
-    - brew update
-    - echo libtool autoconf automake pkg-config nghttp2 | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+    - echo libtool autoconf automake pkg-config ${install_packages} | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
     - brew update && for i in 1 2 3; do brew bundle install --no-lock --file /tmp/Brewfile && break || sleep 1; done
+    - export PKG_CONFIG_PATH=${prefix}
```

Comment:
Try `$(eval echo $prefix)` instead of just `${prefix}`

Reply:
:/ doesn't work